python-avd/pyavd/_eos_designs/schema/schema_fragments/platform_settings.schema.yml

```
@@ -131,6 +151,9 @@ keys:
               type: bool
               default: false
               description: Support for all-active EVPN gateway redundancy.
+            hardware_counter_feature:
```

Avoid adding `_feature` since we are already under `feature_support`.
```suggestion
            hardware_counters:
```

---

this `feature` in this case is the actual CLI key

```
veos-4-33-0f(config-s-sac)#hardware counter ?
  feature  Counter feature
veos-4-33-0f(config-s-sac)#
```

---

we don't need to follow CLI here. Also this may cover more CLI in the future.

---

What we are tracking here are platform capabilities, and the capability is hardware counters.

---

sure, renamed to `hardware_counters`